### DISCUSSION TITLE
我的世界 基岩版|官方中文|V1.20.4|解压即撸

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1453] [其他]
![截图20240811185002](https://github.com/user-attachments/assets/6d32dbcf-1fef-44de-9700-6e6085701ac1)

![](https://www.minecraft.net/content/dam/games/minecraft/screenshots/Allay.png)
![](https://www.minecraft.net/content/dam/games/minecraft/screenshots/Mangrove_House.png)
![](https://www.minecraft.net/content/dam/games/minecraft/screenshots/Mangrove_Biome_2.png)
![](https://www.minecraft.net/content/dam/games/minecraft/screenshots/Frogs.png)
![](https://www.minecraft.net/content/dam/games/minecraft/screenshots/Warden_2.png)
MINECRAFT JAVA EDITION

《我的世界 Minecraft 》是瑞典游戏公司Mojang制作的一款沙盒类游戏，

游戏最初由瑞典人马库斯·阿列克谢·泊松（Markus ‘Notch’ Persson）单独开发。

这个游戏让每一个玩家在三维空间中使用方块这一基础单位来搭建不同的建筑，

展现创意的时刻就此到来，如果乐高太贵又缺乏互动，何不来《我的世界》创造出属于自己的小屋呢？!

这个游戏没有终极目标，只有发挥你无限的想象力。

 
链接: https://pan.baidu.com/s/1rRIClb9rrG7Zg7g-0cuFXw

提取码: qdrc

解压密码:laoquzhang.com

**Comment by lllliufeng** [comment]
布什·戈门

**Comment by lllliufeng** [comment]
《基岩版》《JAVA EDITION》
虽然我买了，还是忍不住吐槽

**Reply by EcoFur** [reply]
我Java、Windows是分开买的，花了200多，之前打折一起才40………………哭死

**Comment by Magic-gif** [comment]
还有这个版本，一直拿网易版玩的

**Comment by EcoFur** [comment]
………………我160入Java，70入Windows，$6.99入的iOS

**Comment by ei-h9** [comment]
哇哦

**Comment by cpu644** [comment]
> ………………我160入Java，70入Windows，$6.99入的iOS
幽默，我165入win10 后来微软账号似了又补了165 然后合并了又165入了基岩java。吗的这东西已经花我一个3A的钱了

**Reply by EcoFur** [reply]
是的，160其实是我买过最贵的游戏了

**Comment by su2569** [comment]
6

**Comment by SSTV-10068** [comment]
您好，您的委托可以挂在SSTV上的广告位上喔~
SSTV广告位价格实惠，广告醒目吸人，[欢迎大家抢购！](https://github.com/share121/inter-knot/discussions/1065)
